packages/terra-menu/src/MenuItem.module.scss

```diff
@@ -54,6 +54,10 @@
     .chevron {
       color: var(--terra-menu-item-disabled-chevron-icon-color, #9b9fa1);
     }
+
+    .instructionsforuse {
+      color: #9b9fa1;
```

Comment:
Should this be themable? cc/ @neilpfeiffer

Reply:
updated : [here](https://github.com/cerner/terra-framework/blob/bba6e4caf837ac5626979e59328f471eba18c68a/packages/terra-menu/src/MenuItem.module.scss#L58-L60)